Update and extend lazer mod articles (#14794)
ppy/osu-wiki: wiki/Gameplay/Game_modifier_(lazer)/en.md

Author: Can I remove this? Or is there anything else I should add/rewrite?

Reviewer: yeah, go for the removal
I also think removing the "customise" section altogether is a good idea since it doesn't really add anything significant

![](img/mod-select.jpg "Mod selection screen for the osu! game mode")

> **Game modifiers** (*mods* for short) are separated into five categories: `Difficulty Reduction`, `Difficulty Increase`, `Automation`, `Conversion` and `Fun`. Some of them can reduce or increase the Score Multiplier. When multiple mods are used simultaneously, their respective multipliers will be multiplied together (e.g. `1.06x * 1.12x = 1.1872x`). There is a also sixth `System` category, but mods from this category cannot be applied manually.

Reviewer: ```suggestion
**Game modifiers**, or **mods**, are separated into six categories: `Difficulty Reduction`, `Difficulty Increase`, `Automation`, `Conversion`, `Fun` and `System`. Some of them can reduce or increase the score multiplier. When multiple mods are used simultaneously, their respective multipliers will be multiplied together (e.g. `1.06x * 1.12x = 1.1872x`). Mods from the `System` category are applied automatically when necessary.
```
